scripts/cli.rb

```diff
@@ -12,6 +12,7 @@ class Cli
     APPLIANCE_URL = "https://github.com/ManageIQ/manageiq-appliance.git"
     BUILD_URL     = "https://github.com/ManageIQ/manageiq-appliance-build.git"
     SUI_URL       = "https://github.com/ManageIQ/manageiq-ui-service.git"
+    V2V_URL       = "https://raw.github.com/ManageIQ/manageiq-v2v-conversion_host-build"
```

Comment:
This is a bit odd as the others are source an this one is a raw file.

Makes me think, does it make sense for the V2V kickstart file to live in this repo? Why are we building it here while it still exists externally? @Fryguy ?

Reply:
Yeah, I knot it's odd... I thought about cloning first, but I really just need 1 file from there and figured I could just go for a raw file since v2v appliance is different anyway...

Reply:
Maybe don't name it `_URL`, since it's different?  That is, if it were named V2V_KICKSTART, for example, I'm not sure there would have been much of a question because even by name it's "different"

Reply:
> Makes me think, does it make sense for the V2V kickstart file to live in this repo? Why are we building it here while it still exists externally? @Fryguy ?

I'm not sure... This isn't part of the main appliance; it's a totally separate appliance.  I'm on the fence where it lives... from a v2v team autonomy point of view, it's easier over there.

Reply:
@simaishi I know the variable name is getting long, but it's not really a v2v url (when I hear v2v I think of the plugin), but instead this is the conversion host, so maybe V2V_CONV_HOST_KICKSTART ?

Reply:
Changed to V2V_CONV_HOST_KICKSTART